### PR TITLE
CRYP-20: Setup detox system testing framework

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://cryptify.atlassian.net/browse/CRYP-20

### CODE REVIEW

File: packages/client/app.json [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
+      "backgroundColor": "#ffffff"
+    },
+    "updates": {
+      "fallbackToCacheTimeout": 0
+    },
+    "assetBundlePatterns": [
+      "**/*"
+    ],
+    "ios": {
+      "supportsTablet": true
```
Comment (domsec) [inline packages/client/app.json:22]: Should we set this to false since we aren't currently supporting tablet?

**Review by domsec — APPROVED** [review]
Looks good.

File: packages/client/app.json (hunk 1 above)
Comment (jason-gerard) [reply packages/client/app.json:22]: Yep there is a lot of other config we will do in this file for iOS and android once we start loading it on actual devices and seeing how it works, couple other things I'll do in this pr before merging though, update docs and set up some scripts mostly

Comment (domsec) [reply packages/client/app.json:22]: Sounds good.

**Review by Andre-Ibrahim — APPROVED** [review]
very nice

**Pull request merged**